src/Validation/Validator.php

```diff
@@ -30,6 +30,12 @@
  */
 class Validator implements ArrayAccess, IteratorAggregate, Countable
 {
+    /**
+     * Used to flag nested rules created with addNested() and addNestedMany()
+     *
+     * @var string
+     */
+    const NESTED = '_nested';
```

Comment:
Generally we try to keep `_` prefixed things as internal use. We do this in the router as well.